src/pages/MyToDoApp.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import TodoList from "../TodoList";
+import AddTodoForm from "../AddTodoForm";
+
+// create custom hook
+// function useSemiPersistentState() {
+//   // Create new state variable named todoTitle with setter setTodoTitle
+//   const [todoList, setTodoList] = React.useState(
+//     JSON.parse(localStorage.getItem("savedTodoList")) || []
+//   );
+
+//   // Define a useEffect React hook with todoList as a dependency
+//   // Inside the side-effect handler function, save the todoList inside localStorage with the key "savedTodoList"
+//   // convert todoList to a string before saving in localStorage
+//   React.useEffect(() => {
+//     localStorage.setItem("savedTodoList", JSON.stringify(todoList));
+//   }, [todoList]);
+
+//   return [todoList, setTodoList];
+
+// }
+
+const API_ENDPOINT = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default/`;
+
+function MyToDoApp() {
+  const [todoList, setTodoList] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  React.useEffect(() => {
```

Comment:
You could consider importing your state and effect hooks at the top e.g `import React, { useState, useEffect }`, by doing so you don't have to repeat `React.useState([]};` instead the following lines of code will look like this `const [todoList, setTodoList] = useState([]);` `const [isLoading, setIsLoading] = useState(true);` `useEffect(() => {}`

Reply:
@KingsleyIbe Thank you for your review and suggestion. I definitely will use importing my state. Do I have to change my pull request now or can I do it in the next lessons?

Reply:
You can effect these changes in the next update/lessons.